starter-project-web-react/pages/blogs.tsx

```diff
@@ -0,0 +1,50 @@
+
+const Bloglist = () => {
+    const blogs = [
+              {
+                "title": "My First Blog",
+                "body" : "Lorem ipsum dolor sit amet consectetur adipisicing elit. Assumenda a rerum quibusdam eligendi doloribus esse aspernatur sit dolorum? Id perferendis excepturi officia quia. Beatae porro natus aliquam deleniti perferendis illum!",
+                "author": "mario",
+                "id": 1
+              },
+              {
+                "title": "Opening Party!",
+                
+                "body" : "Lorem ipsum dolor sit amet consectetur adipisicing elit. Assumenda a rerum quibusdam eligendi doloribus esse aspernatur sit dolorum? Id perferendis excepturi officia quia. Beatae porro natus aliquam deleniti perferendis illum!",
+                "author": "yoshi",
+                "id": 2
+              },
+              {
+                "title": "Grand kalab",
+                "body": "is a truly amazing master of all",
+                "author": "Kalab",
+                "id": 12
+              },
+              {
+                "title": "good man!",
+                "body" : "Lorem ipsum dolor sit amet consectetur adipisicing elit. Assumenda a rerum quibusdam eligendi doloribus esse aspernatur sit dolorum? Id perferendis excepturi officia quia. Beatae porro natus aliquam deleniti perferendis illum!",
+                "author": "yoshi",
+                "id": 2
+              },      
+              {
+                "title": "good man!",
+                "body" : "Lorem ipsum dolor sit amet consectetur adipisicing elit. Assumenda a rerum quibusdam eligendi doloribus esse aspernatur sit dolorum? Id perferendis excepturi officia quia. Beatae porro natus aliquam deleniti perferendis illum!",
+                "author": "yoshi",
+                "id": 2
+              }, 
+        ]
+    return (  
+        <div className="blog-list">
```

Comment:
Is this using material UI or just plain CSS?

Reply:
plain css for now, but i am going to look into that.